src/js/components/ServiceDetail.js

```diff
@@ -67,13 +75,22 @@ class ServiceDetail
             {this.tabs_getUnroutedTabs()}
           </ul>
         </div>
-        {this.tabs_getTabView()}
+        <div className="
```

Comment:
Not sure if the line breaks are correct here?
Are they?

The dangling `"`  looks strange.